x-pack/plugins/security_solution/server/usage/detections/detection_rule_helpers.ts

```diff
@@ -282,7 +323,7 @@ export const getDetectionRuleMetrics = async (
           rule_name: hit._source?.alert.name,
           rule_id: hit._source?.alert.params.ruleId,
           rule_type: hit._source?.alert.params.type,
-          rule_version: hit._source?.alert.params.version,
+          rule_version: Number(hit._source?.alert.params.version),
```

Comment:
NOTE: Now that `rule_version` is typed here we have to convert from `string` to `number` since the telemetry wants this as a number but the alert parameters have had this as a string.